<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>客户端</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QStackedWidget" name="stackedWidget_main">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_4">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>178</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>83</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>用户名：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_username">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>密码：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_pd">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>30</height>
               </size>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_loginin">
              <property name="text">
               <string>登陆</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>83</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>178</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_5">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="titleLabel">
            <property name="styleSheet">
             <string notr="true">#titleLabel{
background: white;
border: none;
border-bottom: 1px solid black;
padding: 5px;
font: 24pt&quot;黑体&quot;;
color: rgb(255,255,255);
background-color:rgba(0,170,127,255);
}
#mainFrame {
border: none;
background: white;}
#loginFrame {
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #ddf, stop: 1 #aaf);
border: 1px solid gray;
padding: 10px;
border-radius: 25px;}</string>
            </property>
            <property name="text">
             <string>欢迎来到聊天室</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="loginPage">
             <layout class="QGridLayout" name="gridLayout">
              <item row="1" column="0">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>31</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="1" colspan="2">
               <widget class="QFrame" name="loginFrame">
                <property name="styleSheet">
                 <string notr="true">#titleLabel{
background: white;
color: rgb(255,255,255);
font-size: 20px;
border: none;
border-bottom: 1px sold black;
padding: 5px;}
#mainFrame {
border: none;
background: white;}
#loginFrame {
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #ddf, stop: 1 #aaf);
border: 1px solid gray;
padding: 10px;
border-radius: 25px}</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout">
                     <item>
                      <widget class="QLabel" name="label_2">
                       <property name="styleSheet">
                        <string notr="true">#label_2{
color:black;
}</string>
                       </property>
                       <property name="text">
                        <string>服务器地址:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="serverEdit">
                       <property name="styleSheet">
                        <string notr="true">#serverEdit{
background:white;
color:black;
}
</string>
                       </property>
                       <property name="text">
                        <string>127.0.0.1</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                     <item>
                      <widget class="QLabel" name="label">
                       <property name="styleSheet">
                        <string notr="true">#label_3{
color:black;
}</string>
                       </property>
                       <property name="text">
                        <string>       昵称    : </string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="userNameEdit">
                       <property name="styleSheet">
                        <string notr="true">#userNameEdit{
background:white;
color:black;
}</string>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <spacer name="verticalSpacer">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Fixed</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>10</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="loginButton">
                   <property name="text">
                    <string>登录聊天室</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="1" column="3">
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>32</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="2" column="2">
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>83</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="1">
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>148</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="chatPage">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QTextEdit" name="roomTextEdit">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QListWidget" name="userListWidget">
                  <property name="maximumSize">
                   <size>
                    <width>120</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_kickout">
                  <property name="text">
                   <string>踢出</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLineEdit" name="sayLineEdit"/>
                </item>
                <item>
                 <widget class="QPushButton" name="sayButton">
                  <property name="text">
                   <string>发送</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="loginoutButton">
                  <property name="text">
                   <string>退出</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_privatename">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QTextBrowser" name="textBrowser"/>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLineEdit" name="lineEdit_private"/>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_privatesend">
                  <property name="text">
                   <string>发送</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_privatereturn">
                  <property name="text">
                   <string>返回大厅</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
